.github/workflows/it.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           node-version: '16'
       - name: build docker images
-        run: ./build-docker-images.sh
+        run: ./build-docker-images.sh "devnet"
```

Comment:
Where are we going to publish the image?

Reply:
So far we haven't published any images yet. We can create a Github action to publish all images with git tags to our docker hub account if unit test and integration tests pass. Images that are different among different networks are bridge_ui & explorer.

Another approach is to extract the configs and use environment variables instead. WDTY?

Reply:
1. I have discussed with Muchen to rework the whole config mess 😅
2. I cannot compile the project with my Apple m1 laptop. Will later run it on my Linux machine and give more feedback to this.